scale/docs/architecture/jobs/resources.rst

```diff
@@ -0,0 +1,54 @@
+
+.. _architecture_jobs_resources:
+
+Resources
+========================================================================================================================
+
+The resources document is a JSON document that defines a set of named resources with floating point values.
+
+**Example job configuration:**
+
+.. code-block:: javascript
+
+  {
+     "version": "1.0",
+     "resources": {
+        "cpus": 1.5,
```

Comment:
Yeah, that's the idea. I don't know how we'll want to treat custom resources in Seed (which will impact our job types).